src/stronghold/mod.rs

```diff
@@ -162,57 +168,61 @@ impl StrongholdAdapterBuilder {
         if let (Some(key), Some(Some(timeout))) = (&self.key, self.timeout) {
             let timeout_task = Arc::new(Mutex::new(None));
 
+            // Stronghold isn't optional, so we must initialize one here if it hasn't been supplied.
+            let stronghold = self.stronghold.unwrap_or_else(|| {
+                // XXX: we unwrap here.
```

Comment:
Weird comment?

Reply:
It's for the next line; we're still on an older stable version of Stronghold, so we initialize `riker::ActorSystem` below, which returns a `Result`, but the `build()` method is not supposed to fail, so an `unwrap()` is left here.

It's actually a copy-pasta from the `Default` implementation (L118). To remove unwraps we can remove the `impl Default` and change `build() -> StrongholdAdapter` to `try_build() -> Result<StrongholdAdapter>`.

Reply:
Yeah but I mean, this comment is basically useless. We can see that we are unwrapping, we would rather be expecting an explanation of why we are 100% this is not going to panic. And replace XXX with `PANIC`.

Reply:
Had a broader discussion and unwraps will be removed.